alerting/build.gradle

```diff
@@ -29,6 +30,29 @@ opensearchplugin {
     extendedPlugins = ['lang-painless']
 }
 
+publishing {
+    publications {
+        pluginZip(MavenPublication) { publication ->
+            pom {
+                name = "opensearch-alerting"
+                description = "OpenSearch Alerting plugin"
+                licenses {
+                    license {
+                        name = "The Apache License, Version 2.0"
+                        url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+                    }
+                }
+                developers {
+                    developer {
+                        name = "OpenSearch"
+                        url = "https://github.com/opensearch-project/ml-commons"
```

Comment:
Should this be `OpenSearch` link instead of `ml-commons`?

Reply:
good catch @qreshi it should be the pluginName.